src/combinatorics/simple_graph/order.lean

```diff
@@ -0,0 +1,79 @@
+/-
+Copyright (c) 2021 Eric. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez
+-/
+import combinatorics.simple_graph.subgraph
+
+/-!
+# Order relation on simple graphs
+
+This file gives `simple_graph` a `bounded_lattice` instance under the `is_subgraph` relation, and
+proves a couple lemmata about how this relation interacts with `simple_graph.subgraph.spanning_coe`.
+
+# Main definitions:
+
+* `simple_graph.is_subgraph`: A `simple_graph` is a subgraph of another if its `adj` is dominated by
+  the other graph's.
+
+* `bounded_lattice` instance: With this definition, `simple_graph` forms a `bounded_lattice`.
+
+* `simple_graph.to_subgraph`: If a `simple_graph` is a subgraph of another, then you can turn it
+  into a member of the larger graph's `simple_graph.subgraph` type.
+
+-/
+
+namespace simple_graph
+
+variable {V : Type*}
+
+/-- The relation that one `simple_graph` is a subgraph of another. -/
+def is_subgraph (x y : simple_graph V) : Prop := ∀ ⦃v w : V⦄, x.adj v w → y.adj v w
```

Comment:
Do the `⦃...⦄` implicits help here? I ended up removing them in `subgraph.is_subgraph`, but probably for no good reason.  (It's funny that I'm now reciprocating your question from the subgraph PR :smile:)

Reply:
it's weird how soon my review comments all end up being wrong :b Eric (Wieser, not me) suggested them so that I don't have to pepper the file with `@h` for `h : x ≤ y`, because otherwise they got unfolded too eagerly

Reply:
If they're helpful here, would you mind changing `subgraph.is_subgraph` to be consistent?